decidim-ai/README.md

```diff
@@ -0,0 +1,55 @@
+# Decidim::Ai
+
+The Decidim::Ai is a library that aims to provide Artificial Intelligence tools for Decidim. This plugin has been initially developed aiming to analyze the content and provide spam classification using Naive Bayes algorithm.
+All AI related functionality provided by Decidim should be included in this same module.
```

Comment:
Add link to the docs final URL (we can do this after this is merged, but adding the comment so I don't forget about it)

Reply:
Fixed in 85481f86e9df65124297e5a61dcba2296576264c